lib/config-generator.js

```diff
@@ -331,11 +331,31 @@ ConfigGeneraror.prototype = {
 
         return new Promise(function(resolve, reject) {
             var result = [];
-
+            var isRequire = false;
             jsstana.traverse(ast, function(node) {
-                var match = jsstana.match('(or (call define ?) (call define ? ?) (call define ? ? ?))', node);
+                var match = jsstana.match('(or (call define ?) (call define ? ?) (call define ? ? ?) (call ?.define ?) (call ?.define ? ?) (call ?.define ? ? ?) (call require ?) (call require ? ?) (call require ? ? ?) (call ?.require ?) (call ?.require ? ?) (call ?.require ? ? ?))', node);
```

Comment:
Hey @julien, @ipeychev, it's true that this probably doesn't happen in Liferay Portal. The problem I see is that this is perfectly valid code, not necessarily transpiled:

```javascript
let MyDictionary = {
    define: function(word, description) { // Add a word definition to the dictionary }
};

MyDictionary.define('foobar', 'The terms foobar, or foo and others are used as placeholder names (also referred to as metasyntactic variables) in computer programming or computer-related documentation');
```

I'm assuming that running the tool over such a file will actually break it, so this makes me think that we want to only target pure `require` and `define` calls.